static/js/server_events_dispatch.js

```diff
@@ -44,9 +44,13 @@ exports.dispatch_normal_event = function dispatch_normal_event(event) {
 
     case 'reaction':
         if (event.op === 'add') {
-            reactions.add_reaction(event);
+            if (event.user.id !== page_params.user_id) {
```

Comment:
We need to take into account that you may have two windows open, so we should remove this short circuiting logic here.  Instead, we want `add_reaction` and `remove_reaction` to be idempotent and properly handle reactions that are already there.  They probably already work like that, but I'm not sure.  It's possible there's a tiny bit of flicker when this happens, but if it's not annoying, that's fine.  If it is annoying, we can do something to detect that reactions aren't really changing and try to avoid a redraw.